contracts/test/ScheduleTestHarness.sol

```diff
@@ -1,7 +1,7 @@
 // SPDX-License-Identifier: MIT
 pragma solidity =0.8.15;
 
-import {ScheduleUtils, Schedule} from "./ScheduleUtils.sol";
+import {ScheduleUtils, Schedule} from "../ScheduleUtils.sol";
```

Comment:
Wonder if libraries can also be upgraded? could be nice to use [solidity global time units](https://docs.soliditylang.org/en/v0.8.15/units-and-global-variables.html#time-units)

Reply:
I remember you pointed this out before but I wasn't clear where it was relevant since this library only deals in seconds anyway.